src/webservices.js

```diff
@@ -8,21 +8,27 @@ function fetchJson(url, options) {
 }
 
 
-async function parseJsonResponse(response) {
-  const data = await response.json()
-  if (response.status >= 200 && response.status < 300) {
-    return data
-  } else {
-    throw new Error(JSON.stringify(data.error))
-  }
+function parseJsonResponse(response) {
```

Comment:
That was taken from one of the many tutorials about JS Promises... I reckon it's over-hype, but that's for better composability I think....